readme.md

```diff
@@ -32,7 +32,7 @@ Come over to [Gitter](https://gitter.im/klauscfhq/taskbook) or [Twitter](https:/
 - Search & filter items
 - Archive & restore deleted items
 - Lightweight & fast
-- Data written atomically to storage
+- Data written automatically to storage
```

Comment:
`atomically` as in [`atomicity`](https://en.wikipedia.org/wiki/Atomicity_(database_systems))